src/components/UI/Cards/ProductCardWithPrice.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div :id="name | toId">
+        <div class="flex w-full h-full justify-center items-center">
+            <div class="flex flex-col w-2/5 h-auto justify-center items-center cursor-pointer">
```

Comment:
When your work with width you should add specific width like w-64 or any another width classes
or use max-w- classes like max-w-sm and give it w-full